compiler/hsSyn/HsTypes.hs

```diff
@@ -403,13 +404,15 @@ instance OutputableBndr HsIPName where
 data HsTyVarBndr pass
   = UserTyVar        -- no explicit kinding
          (Located (IdP pass))
+         ArgFlag
         -- See Note [Located RdrNames] in HsExpr
   | KindedTyVar
          (Located (IdP pass))
          (LHsKind pass)  -- The user-supplied kind signature
         -- ^
         --  - 'ApiAnnotation.AnnKeywordId' : 'ApiAnnotation.AnnOpen',
```

Comment:
Aha, I was wondering about that. I'll add a TODO to the source making sure we don't forget. And I'll also tag all my personal TODOs with KWF so that we don't get them mixed up.